content/docs/installation/kubectl.md

```diff
@@ -117,6 +117,11 @@ First, delete existing cert-manager webhook configurations, if any:
 kubectl delete mutatingwebhookconfigurations cert-manager-webhook
 ```
 
+Then try do delete the corresponding validatingwebhookconfigurations
+```bash
+kubectl delete validatingwebhookconfigurations cert-manager-webhook
```

Comment:
Could we just move this line to the already existing code block under the line that deletes mutating configurations?

Reply:
Sure

Reply:
done